src/System.Runtime.InteropServices.WindowsRuntime/tests/System/Runtime/InteropServices/WindowsRuntime/WindowsRuntimeMarshalTests.cs

```diff
@@ -281,25 +281,25 @@ public void StringToHString_PtrToHString_ReturnsExpected(string s)
             }
         }
 
-        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsWinRTSupported))]
+        [ConditionalFact("System.PlatformDetection, CoreFx.Private.TestUtilities!" + nameof(PlatformDetection.IsWinRTSupported))]
```

Comment:
We may need to use ConditionalXXXX with other assemblies. shorten it the way you proposed, we'll have to do some heuristics handling it to make it work with all scenarios.